module/zfs/dmu_redact.c

```diff
@@ -816,6 +816,7 @@ perform_thread_merge(bqueue_t *q, uint32_t num_threads,
 		avl_remove(&end_tree, &redact_nodes[i]);
 		kmem_free(redact_nodes[i].record,
 		    sizeof (struct redact_record));
+		bqueue_destroy(&thread_args[i]->q);
```

Comment:
```
 ../../module/zfs/dmu_redact.c: In function ‘perform_thread_merge’:
../../module/zfs/dmu_redact.c:819:33: error: invalid type argument of ‘->’ (have ‘struct redact_thread_arg’)
   bqueue_destroy(&thread_args[i]->q);
```

We got a build failure here, it looks to me like you want:

```suggestion
		bqueue_destroy(&thread_args[i].q);
```